include/ableton/link/HostTimeFilter.hpp

```diff
@@ -28,7 +28,7 @@ namespace ableton
 namespace link
 {
 
-template <class T>
+template <class ClockT>
```

Comment:
I like the idea of renaming this template parameter. For consistency reasons I suggest to call it `Clock` instead of `TClock`.
See for example: https://github.com/Ableton/link/blob/60adcc562a284cf76f7449f5ea1be155680b8dda/include/ableton/link/Measurement.hpp#L38